torchtitan/experiments/forge/example_train.py

```diff
@@ -0,0 +1,351 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
Skipping over reviewing this file since I assume it's meant more as a demonstration and not really intended to be used directly. Lmk if it warrants a closer look though